cli/auth/auth_command.py

```diff
@@ -18,6 +35,37 @@ def authenticate(context: click.Context):
         _handle_exception(context, e)
 
 
+@authenticate.command(name='check')
+@click.pass_context
+def authorization_check(context: click.Context):
+    """ Check your machine associating CLI with your cycode account """
+    def print_result(*, result: bool) -> None:
+        # the current impl of printers supports only results of scans
+        if context.obj['output'] == 'text':
+            secho_args = {
```

Comment:
what about something like:
`failed_auth_check_args = {'message': 'You are not authorized', 'fg': 'red'}`
`passed_auth_check_args = {'message': 'You are not authorized', 'fg': 'red'}`
`auth_check_result = passed_auth_check_args if check_passed else failed_auth_check_args`
`return click.secho(**auth_check_result)`